server/schedulers/hot_region.go

```diff
@@ -228,7 +230,7 @@ func (h *hotScheduler) gcRegionPendings() {
 		empty := true
 		for ty, op := range pendings {
 			if op != nil && op.IsEnd() {
-				if time.Now().After(op.GetCreateTime().Add(minRegionScheduleInterval)) {
+				if time.Now().After(op.GetCreateTime().Add(h.conf.GetMaxZombieDuration())) {
```

Comment:
GetMaxZombieDuration？

Reply:
Yes, it still influences the store, which should be consistent with the `pendingSum`.